src/config.h

```diff
@@ -40,8 +40,12 @@
 #include <fcntl.h>
 #endif
 
+#if defined(__APPLE__) && defined(__MAC_OS_X_VERSION_MAX_ALLOWED) && __MAC_OS_X_VERSION_MAX_ALLOWED >= 1060
+#define MAC_OS_10_6_DETECTED
+#endif
+
```

Comment:
Please add this to `fmacro.h`

Reply:
I just thought about it too, but `__MAC_OS_X_VERSION_MAX_ALLOWED` is defined after `#include <AvailabilityMacros.h>`.

Reply:
Yes, you are right, let's keep it as now.